ledger/ledger-api-common/src/test/suite/scala/com/digitalasset/platform/server/api/services/grpc/GrpcHealthServiceSpec.scala

```diff
@@ -31,24 +29,269 @@ final class GrpcHealthServiceSpec
   private implicit val executionContext: ExecutionContext = materializer.executionContext
 
   "HealthService" should {
-    "check the current health" in {
-      val service = new GrpcHealthService()
+    "report SERVING if there are no health checks" in {
+      val service = new GrpcHealthService(new HealthChecks)
+
+      val response = Await.result(service.check(allServicesRequest), patienceConfig.timeout)
+
+      response should be(servingResponse)
```

Comment:
```suggestion
      response shouldBe servingResponse
```
An alternative, if you wish.

Reply:
I prefer `should be(…)` because often I want to use `should equal(…)` or `should contain(…)`, etc. I like them to be uniform. Happy to change them if it's more consistent though.